src/platforms/unity/migration.mdx

```diff
@@ -6,13 +6,14 @@ redirect_from:
   - /platforms/unity/unity-lite/
 ---
 
-## Migrating From 0.3.0 or Earlier
+## Migrating to 0.28.0
```

Comment:
```suggestion
## Migrating to 0.28.0

Here are the notable changes when migrating to version 0.28.0.
```
Wanted to add an introductory line so that it's not 3 headings in a row.